mscore/palette/palettetree.cpp

```diff
@@ -286,9 +286,9 @@ bool PalettePanel::read(XmlReader& e)
                   e.skipCurrentElement();
             else if (t == "type") {
                   bool ok;
-                  const int t = QMetaEnum::fromType<Type>().keyToValue(e.readElementText().toLatin1().constData(), &ok);
+                  const int t_ = QMetaEnum::fromType<Type>().keyToValue(e.readElementText().toLatin1().constData(), &ok);
```

Comment:
`type` or `localType` is better, IMHO. Using underline as an ending of a variable is a bad practice.

Reply:
bit also longer. Global or outer types should have longer and more meaningful names, the more local/inner, the shorter they can get, as then their definition is within sight.
I could make it `t1` here or some such?

Reply:
I don't like numbers in variable names (as well as world C++ experts) :) It's better to rename the outer one.

Reply:
t1 is being used further up in the code, same file

Reply:
Hmm, here I guess `s` for the outer (being a string or rather a `QStringRef`) and `t`for the inner (being an int or `Type`) should be OK. Such a change is more invasive though, I try to keep those warning fixes as small as possible.

Reply:
Looking at some ofg my earlier PRs fixing warnings I found some more 'sins' of that sort.
Should I go back and fix those too?
One such case is in #4963, here we'd bettert change musescore.h `ScriptEngine* se     { 0 };`  to `ScriptEngine* scriptEngine     { 0 };` or even remove that apparently unused member alltogether